compiler/fir/fir2ir/src/org/jetbrains/kotlin/fir/backend/Fir2IrVisitor.kt

```diff
@@ -77,6 +77,14 @@ class Fir2IrVisitor(
         TODO("Should not be here: ${element.render()}")
     }
 
+    override fun visitField(field: FirField, data: Any?): IrField {
+        if (field.origin == FirDeclarationOrigin.Synthetic) {
+            return declarationStorage.getCachedIrField(field)!!
+        } else {
+            TODO("Unexpected field: ${field.render()}")
```

Comment:
This (and above one too) looks like an assertion error, not `TODO`. :)

Reply:
done